src/FormManager.php

```diff
@@ -432,6 +437,20 @@ private function prepare(): void
             }
         }
 
+        // If the last form field is not a page break, we need to merge the last step into the previous one
+        if (!$isPageBreakLastFormField) {
+            $lastStepIndex = count($this->formFieldsPerStep) - 1;
+
+            if (isset($this->formFieldsPerStep[$lastStepIndex], $this->formFieldsPerStep[$lastStepIndex - 1])) {
+                $this->formFieldsPerStep[$lastStepIndex - 1] = [
+                    ...$this->formFieldsPerStep[$lastStepIndex - 1], // Second last step
+                    ...$this->formFieldsPerStep[$lastStepIndex], // Last step
+                ];
+
+                unset($this->formFieldsPerStep[$lastStepIndex]);
```

Comment:
Looks like we would also need an `array_values()` to ensure that the indexes of the array are re-ordered, right?

Reply:
I don't think so, this is a non-associative array and we just drop the very last step.

Reply:
Then we should use `array_pop()` to make it clear.